### PR TITLE
chore: fixes #66

### LINKED ISSUE #66: test runner doesn't work with the printer
I tried to parse an simple identifier "aladdin". It doesn't get printed in the '.md'. However. If I manually return a text string it get printed.

My guess is that there are no CST node given to the printer, or the CST node is wrong?

### PR DESCRIPTION
**Pull request by KFlash**

### CODE REVIEW

**Comment by KFlash** [comment]
@aladdin-add  It works locally after I did this change.  If this is ok, just refresh the snapshots and merge.

File: test/runner/tob.ts [hunk 1]
```diff
@@ -120,7 +120,7 @@ function outputBlock(tob: Tob, updateItems: any) {
   return `
 ## Output
 
-### Hybrid CST
+### CST
```
Comment (aladdin-add) [inline test/runner/tob.ts:123]: why changing this?

**Review by aladdin-add — CHANGES_REQUESTED** [review]

File: test/runner/tob.ts (hunk 1 above)
Comment (KFlash) [reply test/runner/tob.ts:123]: It's not an hybrid CST.

Comment (aladdin-add) [reply test/runner/tob.ts:123]: you have to change the read logic - if want to do this.

Comment (KFlash) [reply test/runner/tob.ts:123]: can you fix that and merge?

Comment (aladdin-add) [reply test/runner/tob.ts:123]: and you also need to update all the md files. or the tests will be failing.

I would say, better to do it in another PR - one PR for one thing.

Comment (KFlash) [reply test/runner/tob.ts:123]: okay. I'm reverting it and let you look into it.

**Review by aladdin-add — APPROVED** [review]

**Pull request merged**